package.json

```diff
@@ -28,7 +29,7 @@
         "@typescript-eslint/parser": "^6.21.0",
         "agadoo": "^3.0.0",
         "eslint": "^8.57.0",
-        "eslint-config-turbo": "^1.13.3",
+        "eslint-config-turbo": "^2.0.3",
```

Comment:
Are these package version bumps also necessary?

Reply:
Uh. Wtf. These are already in. Maybe I didn't rebase this PR…

Reply:
There. Rebased. That makes more sense.